### ISSUE TITLE
Cmake shows error after running make

### ISSUE THREAD

**Issue by tea0112** [fcitx/fcitx5-bamboo#10]
`➜ git submodule update --init

Submodule 'bamboo/bamboo-core' (https://github.com/BambooEngine/bamboo-core) registered for path 'bamboo/bamboo-core'

Cloning into '/home/user/workspace/fcitx5-bamboo/bamboo/bamboo-core'...

Submodule path 'bamboo/bamboo-core': checked out 'f0ca534819fa84ee37df9277ff2449d82ed1ec3e'

fcitx5-bamboo on  main via △ v3.29.5

➜ mkdir -p build

fcitx5-bamboo on  main via △ v3.29.5

➜ cd build

fcitx5-bamboo/build on  main

➜ cmake ../ -DCMAKE_INSTALL_PREFIX=/usr

-- The C compiler identification is GNU 11.4.0

-- The CXX compiler identification is GNU 11.4.0

-- Detecting C compiler ABI info

-- Detecting C compiler ABI info - done

-- Check for working C compiler: /usr/bin/cc - skipped

-- Detecting C compile features

-- Detecting C compile features - done

-- Detecting CXX compiler ABI info

-- Detecting CXX compiler ABI info - done

-- Check for working CXX compiler: /usr/bin/c++ - skipped

-- Detecting CXX compile features

-- Detecting CXX compile features - done

CMake Error at CMakeLists.txt:11 (find_package):

  By not providing "FindFcitx5Core.cmake" in CMAKE_MODULE_PATH this project

  has asked CMake to find a package configuration file provided by

  "Fcitx5Core", but CMake did not find one.

  Could not find a package configuration file provided by "Fcitx5Core"

  (requested version 5.0.22) with any of the following names:

    Fcitx5CoreConfig.cmake

    fcitx5core-config.cmake

  Add the installation prefix of "Fcitx5Core" to CMAKE_PREFIX_PATH or set

  "Fcitx5Core_DIR" to a directory containing one of the above files.  If

  "Fcitx5Core" provides a separate development package or SDK, be sure it has

  been installed.

-- Configuring incomplete, errors occurred!`

**Comment by hajuha** [comment]
got the same error

**Comment by goatastronaut0212** [comment]
I don't think this looks like a valid issue somehow. You may want to research how Nixpkgs or Arch PKGBUILD build it.
